### PR TITLE
Refactor openedx plugin_api to web-fragments

### PR DESCRIPTION
**Pull request by wowkalucky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Calendar may be shared by stuff personal with provided google accounts
     . edxapp_env
     pip install -e git+https://github.com/raccoongang/edx-calendar-tab.git@master#egg=edx-calendar-tab
 
+Django's collectstatic should be performed.
```
Comment (z4y4ts) [inline README.md]: Add sample command

File: calendar_tab/views.py [hunk 2]
```diff
@@ -13,14 +13,16 @@
 from opaque_keys.edx.keys import CourseKey
 from courseware.access import has_access
 from courseware.courses import get_course_with_access
-from openedx.core.djangoapps.plugin_api.views import EdxFragmentView
+# from openedx.core.djangoapps.plugin_api.views import EdxFragmentView
```
Comment (z4y4ts) [inline calendar_tab/views.py]: Don't leave commented out code.

**Review by z4y4ts — COMMENTED** [review]
Please describe briefly the changes in this PR. What and why.

**Comment by wowkalucky** [comment]
Pull request is created to make codebase suitable for changes happened in edx-platform (open-release/ficus.3 versus open-release/ficus.2):
edx-platform/openedx/core/djangoapps/plugin_api/ became deprecated and completely was removed to "web-fragments" standalone package.

File: requirements.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@
 # Django and openEdx's edx-platform are the runtime for this app.
 
 google-api-python-client==1.6.2
+web-fragments==0.2.2
```
Comment (z4y4ts) [inline requirements.txt:5]: Is `web-fragments` installed into devstack/fullstack?
Anyways, add it to `setup.py`

**Review by z4y4ts — APPROVED** [review]
Looks good in general, but add the dependency to `setup.py` before merging.

**Comment by z4y4ts** [comment]
And of course, fix codeclimate issues.

File: requirements.txt (hunk 3 above)
Comment (z4y4ts) [reply requirements.txt:5]: ✅

**Pull request merged**